pkg/che/che.go

```diff
@@ -120,6 +120,24 @@ func (c *Client) DeleteUser(userID string) error {
 	return err
 }
 
+// UserAPICheck returns an error if the Che user API cannot be reached successfully, returns nil otherwise
+func (c *Client) UserAPICheck() error {
+	reqData := url.Values{}
+	res, err := c.cheRequest(http.MethodGet, cheUserPath, reqData)
+	if err != nil {
+		return errors.Wrapf(err, "che user API check failed")
+	}
+	defer rest.CloseResponse(res)
+	if res.StatusCode == http.StatusOK {
+		return nil
+	}
+	resBody, readError := rest.ReadBody(res.Body)
+	if readError != nil {
+		log.Error(readError, "error while reading body of the che user API check response")
```

Comment:
Shouldn't we return the error message too in this case instead just logging it?

Reply:
I figured the more important thing is the response status. If we couldn't read the body then the body string would just be empty in the error but it's not that important to know reading the body failed?

Reply:
Ah.. I was a bit sleepy when was reviewing the code. It's the error about reading the body.. So, right, we can leave it as is then :)